src/PrestaShopBundle/Twig/TranslationsExtension.php

```diff
@@ -365,6 +378,109 @@ protected function concatenateSubtreeHeader($subdomain, $subtree, $level = 2)
 
     /**
      * @param $subtree
+     * @param $output
+     * @return string
+     */
+    protected function getTranslationsFormStart(&$subtree, $output)
+    {
+        $id = '';
+        $parentAttribute = ' class="subdomains hide"';
+        if (array_key_exists('__fixed_length_id', $subtree)) {
+            $fixedLengthId = $subtree['__fixed_length_id'];
+            unset($subtree['__fixed_length_id']);
+            $id = ' id="' . $fixedLengthId . '" ';
+            $parentAttribute = ' data-parent-of="' . $fixedLengthId . '"';
+        }
+
+        $domainAttribute = '';
+        if (array_key_exists('__domain', $subtree)) {
+            $domainAttribute = ' data-domain="' . $subtree['__domain'] . '" ';
+            unset($subtree['__domain']);
+        }
+
+        $missingTranslationsAttribute = '';
+        if (array_key_exists('__metadata', $subtree)) {
+            $missingTranslations = $subtree['__metadata']['missing_translations'];
+            $missingTranslationsAttribute = ' data-missing-translations="' . $missingTranslations . '"';
+            unset($subtree['__metadata']);
+        }
+
+        return $this->container->get('templating')->render(
+            'PrestaShopBundle:Admin:Translations/include/translations-form-start.html.twig',
+            array(
+                'id' => $id,
+                'domain' => $domainAttribute,
+                'parent' => $parentAttribute,
+                'missing_translations' => $missingTranslationsAttribute,
+                'title' => $output,
+            )
+        );
+    }
+
+    /**
+     * @param $output
+     * @param $subtree
+     * @return string
+     */
+    protected function replaceWarningPlaceholder($output, $subtree)
+    {
+        $missingTranslationsMessage = '';
+        $missingTranslationsLongMessage = '';
+        $missingTranslationsClass = '';
+        if (array_key_exists('__metadata', $subtree) && $subtree['__metadata']['missing_translations'] > 0) {
+            $missingTranslationsCount = $subtree['__metadata']['missing_translations'];
+            $domain = $subtree['__metadata']['domain'];
+
+            $missingTranslationsMessage =
+                '<div class="missing-translations-short-message pull-right hide">'.
+                $this->translator->trans('{{ translations }} missing',
```

Comment:
why not using %variable% ?
I'm afraid it will be confusing to translators.
I'd use something like %nb_translations%